Alchemy-Pay-questions-1.md

```diff
@@ -0,0 +1,39 @@
+**About Ubiquity** 
+
+Ubiquity is a funded startup building highly automated bounties & freelance marketplace, as well as financial services. 
+
+Our target audience is freelancers - individuals (not businesses) that are located in different countries across the globe. 
+
+We share the Alchemy Pay goal to bridge fiat and crypto global economies seamlessly.
+Ubiquity helps to earn in Web3 and spend everywhere.
+
+Ideally, we are looking for a strategic partner for long-term cooperation that can begin with the branded card issuing.
+
+**1. Do you require any specialized solutions outside of the standard on/off ramping? (e.g. custom coin listing, custom blockchain, bespoke solutions, etc.)**
+
+We need to issue branded Visa or MasterCard cards for freelancers and perform payouts to these cards.
+
+First of all, we want to perform a quick launch with the basic functionality without specialized solutions.
+API for typical actions will be required for this step. 
+As we understand, Alchemy Pay can provide white-label virtual debit  MasterCard cards branded by Ubiquity.  
+
+In the future, we plan to develop capabilities for suitable DeFi management and advanced security. 
+Alchemy Pay off-ramp solution and partnership with DeFi protocols can be a good foundation for these new capabilities. 
+
+**2. What are some important regions or payment methods that you require?**
+
+The goal is to provide a global Ubiquity card. 
+We can start with these regions - Asia (ig Singapore and South Korea),  South America (ig Brazil), and Eastern Europe (ig Poland).
+We want to serve countries where freelancers are located, except all jurisdictions included in the FATF black and gray lists, Russia, Belarus.
```

Comment:
```suggestion
We want to serve countries where freelancers are located.
```

I don't think its beneficial to deliberately list this, as they will simply follow their regulations around compliance.